source/code-snippets/usage-examples/find.js

```diff
@@ -0,0 +1,41 @@
+
+const { MongoClient } = require('mongodb');
+
+
+const client = new MongoClient(
+   'mongodb+srv://admin:admin@mflix-2sp0m.mongodb.net/test?retryWrites=true&w=majority',
+);
+
+async function run() {
+   try {
+      await client.connect();
+
+      const database = client.db('sample_mflix');
+      const collection = database.collection('movies');
+
+      // create a query document for a range match for a movie less than 15 minutes long
```

Comment:
"range match" seems a little jargon-y IMO. Maybe:

`// Query for movies that have a runtime less than 15 minutes`